Makefile

```diff
@@ -1,7 +1,7 @@
 TEST_PATH = ./tests/
 FLAKE8_EXCLUDE = venv,.venv,.eggs,.tox,.git,__pycache__,*.pyc
 PROJECT = pytravharv
-AUTHOR = Cedric_Decruw
+AUTHOR = "Cedric Decruw"#
```

Comment:
change to opsci --> https://github.com/vliz-be-opsci/py-RDF-store/blob/main/Makefile#L4